dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/LogTraceData.java

```diff
@@ -409,7 +428,12 @@ public String getComponent() {
     }
 
     public String getSequence() {
-        return getStringValue(14);
+        String sequenceId = getStringValue(14);
+        if (sequenceId == null || sequenceId.isEmpty()) {
+            sequenceId = sequenceNumber.next(getDatetime());
```

Comment:
the raw sequence number (the value returned from sequenceNumber -- but without the formatting part) needs to be set into the LogTraceData (perhaps in a rawSequenceNumber field).  Same for the date -- though I think we already have the date value in a field.  Here we should just be calling a new method on SequenceNumber to format the raw date and rawSequenceNumber into a formatted sequenceNumber.